docs_user/modules/proc_migrating-ovn-data.adoc

```diff
@@ -2,18 +2,15 @@
 
 = Migrating OVN data
 
-The next step is to migrate data from OVN databases from the original
-{rhos_prev_long} deployment to `ovsdb-server` instances running in the
-{OpenShift} cluster.
+Migrate the data in the OVN databases from the original {rhos_prev_long} deployment to `ovsdb-server` instances that are running in the {OpenShift} ({OpenShiftShort}) cluster.
 
 .Prerequisites
 
-* Make sure the previous Adoption steps have been performed successfully.
- ** The `OpenStackControlPlane` resource must be already created at this point.
- ** `NetworkAttachmentDefinition` CRDs for the original cluster are already defined. Specifically, `internalapi` network is defined.
- ** The original {networking_service} and OVN `northd` are not running.
- ** There must be network routability between control plane services and the adopted cluster.
-* Define the following shell variables. The values that are used are examples. Replace these example values with values that are correct for your environment:
+* The `OpenStackControlPlane` resource is created.
+* `NetworkAttachmentDefinition` custom resources (CRs) for the original cluster are defined. Specifically, the `internalapi` network is defined.
+* Ensure that the original {networking_first_ref} and OVN `northd` are not running.
+* Ensure that there is network routability between the control plane services and the adopted cluster.
+* Define the following shell variables. Replace the example values with values that are correct for your environment:
```

Comment:
There is one more prerequisite which we never mentioned I think and this is that You have to be on the ML2/OVN already. It may be kind of obvious but in OSP 17.1 ML2/OVS is still supported backend so it has to be migrated to ML2/OVN before adoption to RHOSO will be started. I created Jira https://issues.redhat.com/browse/OSPRH-8725 but didn't had time to do it yet.

Reply:
Ok, for now I added the following prerequisite:
The cloud is migrated to the Modular Layer 2 plug-in with Open Virtual Networking (ML2/OVN) mechanism driver.